src/style.css

```
@@ -0,0 +1,19 @@
+body {
+  margin: 0;
+}
+
+div {
```

краще стилізувати через класи, а не теги

---

```suggestion
.block {
```
створи клас, та додай його необхідним елементам